ome/training-scripts
# Analyse idr0021 users meeting 2018

Various scripts to work with images from ```idr0021``` http://idr.openmicroscopy.org/webclient/?show=project-51.

To test:

 - Follow steps in maintenance/idr0021-data-prep.md walkthrough
 - Check that map annotations are added to Images (see below), and Project has bulk_annotations Table.
 - View & filter in OMERO.parade: Filtering by Table ```max_points``` < 50 should show images from most 2 Datasets:

![screen shot 2018-05-07 at 11 54 59](https://user-images.githubusercontent.com/900055/39698681-a407df5a-51ed-11e8-95b2-b87d84bd637d.png)

NB: the channel of interest for filtering shapes is different for every Dataset and even changes several times for images within a Dataset. The channel is shown in the Map Annotation and should refer to the Channel that is named the same as the Dataset.

<img width="347" alt="screen shot 2018-05-02 at 23 34 47" src="https://user-images.githubusercontent.com/900055/39552527-7d1dd032-4e61-11e8-9788-eea0d3a09c7c.png">

TODO: As discussed with @pwalczysko, although we can filter for Shapes on a *single* Channel, we can choose to measure pixel intensity within those shapes for *multiple* channels. Currently all these intensities are averaged together, even for different channels (if we choose to export intensity for multiple channels) but we should keep these separate and report them as ```min_intensity_channel_1```, ```min_intensity_channel_2``` etc.
